### ISSUE TITLE
Transitions does not always work

### ISSUE THREAD

**Issue by Lpwmack17** [vassbo/freeshow#1006]
As the title already says, the transitions betwenn slides and media doesn't work, even though everything is activated.

**Version (Optional)**
 - OS: Windows 11
 - FreeShow Version: 1.3.0

https://github.com/user-attachments/assets/c9cedd4a-869e-4685-be3b-e4b4f9c673d8

**Comment by vassbo** [comment]
https://github.com/ChurchApps/FreeShow/pull/854#issuecomment-2391011258

**Comment by vassbo** [comment]
https://github.com/ChurchApps/FreeShow/issues/995#issuecomment-2466830806

**Comment by vassbo** [comment]
Does one of these comments explain your issue?

**Comment by TPaulManohar** [comment]
@vassbo We tried a lot, in recent version(1.3.1), But the transition is not working as expected, Please looking into this. Every time restart the app in the show, that doesn't make a sense.

Even we kept global transitions that also not working.

**Comment by Lpwmack17** [comment]
@vassbo None of these work, tried in previous versions and the recent ones, but in the transitions don’t work as expected or even not work

**Comment by thebiblelover7** [comment]
I've had the same issue

**Comment by TPaulManohar** [comment]
@vassbo we tried in macOS with v1.3.1 transition are working there.. in windows it not working, And what I identify here is when we created a template with added media & text, transition are not working. It get lagging and transition are not working.

**Comment by vassbo** [comment]
As I mentioned this is most likely a Svelte transition bug, that I can't do anything about until I rewrite the entire transition system. Also as I said for some reason it only happens in production and not in the development environment.

**Comment by PavelS-CZ** [comment]
Same issue here, the transitions dont work for me. Ive tried enabling in menu - nothing
enabling in preview - nothing, enabling in edit slide - nothing. :((((

**Comment by colanct** [comment]
We're also experiencing sporadic global loss of transitions (and have been for a long time), as described in [#854 comment](https://github.com/ChurchApps/FreeShow/pull/854#issuecomment-2391011258 ) on Win 10 & 11 PCs.  Restarting the app usually restores transitions.   As the issue occurs less than 10% of the time, we're unable to provide more details.

The most recent occurrence was on 1.3.6 on a Win10 PC.

**Comment by PavelS-CZ** [comment]
FOUND A SOLUTION WORKING FOR ME, So basically when i tried using transitions just while using only **1 MONITOR** it didn't work, but once I used a **2nd  MONITOR** it projected correctly on it.

**Comment by vreykin** [comment]
I agree, with my setup, when I close a show or verse, it closes rapidly and not consistently and smoothly as when transitioning from one slide-show to another or verse-to-verse. This seems to be a big bug.

Edit: Here's a small visual of what I'm experiencing.

https://github.com/user-attachments/assets/d583aeff-21a9-46a6-923f-6292a00498ad

**Comment by vassbo** [comment]
@Vitlyr You are experiencing a different bug only in 1.4.5 which is fixed in the latest beta!

**Comment by Thomas-Cee** [comment]
Seconding this bug report. Here is me asking about it on the FB group. 

Maybe I am missing something, but I have explored the global transition section, the slide by slide transition, using up to date FreeShow on Mac OS and Windows.

Thank you! 
https://www.facebook.com/691991227/videos/2235685673526926/

**Comment by vassbo** [comment]
@Thomas-Cee Thanks for the video, very helpful. In/out only applies to the slide/item being presented or cleared. So it seems like it uses the globally set "Between" for your list. I think I should add the "Between" option to items as well now as we have line by line reveal!

https://github.com/ChurchApps/FreeShow/issues/944#issuecomment-2443823795
https://github.com/ChurchApps/FreeShow/issues/1245#issuecomment-2636470943

**Comment by Thomas-Cee** [comment]
Thank you @vassbo! Very helpful. I edited my initial question to include an explanation in case anyone else had the same questions.

I uploaded a video of my UPDATED question to FB.

If I should turn that specific question into a feature request I can do that. 

Fantastic software FreeShow is. Thank you so so much for all the work you guys have put into it. It's truly marvelous. The best kept secret in presentation software FOR SURE and we love it so much more than ProPresenter 6 and 7 both of which we used. Ugg. So glad to be done with that bug-ware lol. 
https://www.facebook.com/691991227/videos/1270343241170609/

**Comment by TPaulManohar** [comment]
@vassbo In v1.4.8 Transitions works as expected. From so long we are expecting this.

Thanks.

**Comment by csafarjr** [comment]
I am running v1.4.9 and no matter what transition I set it is always stuck on fade. I noticed that when I switch slides the transition settings on the current style sets itself to Fade. I set all transitions to none and it always changes back to Fade on the style.

**Comment by vassbo** [comment]
@csafarjr Try setting the global transitions (right under the preview on the right), or make sure the style is applied to the output.

**Comment by csafarjr** [comment]
I have transitions set to none is that location as well, on the slides, in
the template, and in the style. But, when I go between slides something is
changing those back to fade.

**Comment by koniygn** [comment]
Thank you for re-directing me to this post. However, can anyone post out the solution again? I digged into the related threads and didn't find any specific resolution. 

In addition, I tried Revo Uninstaller to uninstall FreeShow, and clean up all the leftover files and registry. But a fresh install of 1.5.0 still fails to use Transitions. All slides still blink like PowerPoint, and the same as #1006. Is there anything corrupted outside FreeShow, and what should I do??

Thanks in advance!! 

-Enoch

**Comment by vassbo** [comment]
It's a bit random. You could try toggling off/on the output, or changing the transition duration.

**Comment by vreykin** [comment]
Same thing here, after updating to v1.5.0 I also started experiencing transition issues. It started to become a lot more choppy. Any fixes in the upcoming beta updates?

**Comment by koniygn** [comment]
Don't know if it is useful information, I have a little research and trial-and-error time with ChatGPT, and here is a summary (sorry and please delete if it is not allowed):  

(1) My computer (dated around 2018) is a notebook which cannot be upgraded to Windows 11. 

(2) Its Intel HD 520 (integrated) display card has no new drivers available since 2022, while the AMD R7 M360 (discrete) can still be updated. 

(3) The Intel HD 520 supports only up to WDDM 2.1 (which is my case) but Windows 10 22H2 and 11 would rely on WDDM 3.0 or above (which is unnoticeable in my computer) .

(4) Windows updates around late-September 2025 include some GPU updates (in my case: KB 5066198 and Servicing Stack 10.0.19041.6389) that blocked certain GPU usages for Chromium sandbox-based apps, which is an essential part of FreeShow's architecture. 

(5) My computer cannot switch to "AMD only", and it would considered to be a system error when I disable the Intel display card. 

(6) So I am stuck at WDDM 2.1, which would technically affect certain display functions in my computer's Windows 10 (which is unnoticeable in my computer), and Transition effect inside FreeShow is disabled.

Suggested solution of my case: 

(A) Roll back to Windows 10 21H2 or uninstall recent updates one by one to see at what point Transition effects come back, or

(B) Buy a new computer which uses WDDM > 3.0

---------------------

I am no computer guru, but I hope this explanation help whoever understands. The only concern is, Windows updates would then quietly phase out older computers from using FreeShow, and my church has several computers that is on the edge of having this problem. So... it is technically "phasing out", but a bug at all.

Hope it solves some of the mysteries. God bless!!

-Enoch



-------------

Methods tried but ineffective, prior to the conclusions above: 
(1) Uninstalled and reinstalled 1.4.8, 1.4.9 and 1.5.0
(2) Uninstalled with Revo Uninstaller, which deletes hidden files and registry.
(3) Updated display cards' driver. 
(4) Checked DWM glitches.
(5) Reinstalled DirectX, and rebuilt DirectX shaders and cache.
(6) Disabled hardware acceleration in FreeShow.
(7) Tried the following cmd commands:
[sfc /scannow] 
[DISM /Online /Cleanup-Image /CheckHealth]
[DISM /Online /Cleanup-Image /ScanHealth]
[DISM /Online /Cleanup-Image /RestoreHealth]
(8) Updated .NET Desktop Runtime
(9) Updated Viscual C++ Redistributable

**Comment by vassbo** [comment]
Thanks @koniygn, but the OS version should not make a difference.
